static/js/components/subject_page/block.tsx

```diff
@@ -138,10 +144,30 @@ export function Block(props: BlockPropType): JSX.Element {
             );
           })}
       </div>
+      {isNlInterface() && props.columns.length > NL_NUM_BLOCKS_SHOWN && (
```

Comment:
I'm slightly confused, you mention in the comments of this PR that there is only one column per block .. so why are we checking length of columns?

Can you confirm when the expando should show up and what the layout in terms of tiles/columns/blocks is?